io.github.rocket.editor.yml

```diff
@@ -0,0 +1,34 @@
+app-id: io.github.rocket.editor
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: editor
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=dri
```

Comment:
`- --share=ipc` is also needed.

Reply:
Applied that change, and squashed it all! Thanks!